README.md

```diff
@@ -1,3 +1,23 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Please remove the generated TOC. If we add this, we're likely to use a different generator.